package.json

```diff
@@ -41,6 +41,9 @@
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.7.0",
+    "@ember-data/adapter": "^4.4.3",
+    "@ember-data/model": "^4.4.3",
+    "@ember-data/serializer": "^4.4.3",
```

Comment:
These are peerDependencies of guidemaker that were previously being sloppily provided by ember-data's transitive dependencies. pnpm will more strictly do what you say, and by not having these we were letting guidemaker's "*" dependencies auto-install the very latest `@ember-data/*` packages instead, which don't work.